modules/govuk/manifests/apps/ckan/cronjobs.pp

```diff
@@ -30,8 +33,15 @@
     ensure         => $ensure_solr_reindex,
     paster_command => 'search-index rebuild -o',
     plugin         => 'ckan',
-    hour           => '7',
+    hour           => '4',
     minute         => '0',
   }
 
+  govuk::apps::ckan::paster_cronjob { 'pycsw load':
+    paster_command => "ckan-pycsw load -p /var/ckan/pycsw.cfg -u http://localhost:${ckan_port}",
```

Comment:
There's a variable for `/var/ckan/pycsw.cfg` where this class is used. Could that be made a parameter and used here? That would make it clearer that this class involves using this configuration file.

Reply:
Good idea, I've pushed up an amended commit for this.